client/allocrunner/taskrunner/task_runner.go

```diff
@@ -462,8 +462,29 @@ func (tr *TaskRunner) runDriver() error {
 	// TODO(nickethier): make sure this uses alloc.AllocatedResources once #4750 is rebased
 	taskConfig := tr.buildTaskConfig()
 
-	// TODO: load variables
+	// Build hcl context variables
+	vars, errs, err := tr.envBuilder.Build().AllValues()
+	if err != nil {
+		return err
```

Comment:
Do we want to wrap this error? Looks like it comes directly from `ctyify`. Is there enough context there for it to be useful?

Reply:
Are you saying `running driver failed: missing node variable` might not be a helpful error message?! :stuck_out_tongue_closed_eyes:  Good catch. Adding some context here.

Reply:
The bad news is that _as far as I can tell_ all of the errors would be programming errors (like type assertions failing), so the hopes of additional context helping is slim. Still better to have the context!